examples/esp/esp32s2Saola/ThemeMonoInverseBuilder.h

```diff
@@ -0,0 +1,58 @@
+#ifndef TCMENU_THEME_MONO_INVERSE
+#define TCMENU_THEME_MONO_INVERSE
+
+#include <graphics/TcThemeBuilder.h>
+
+color_t defaultItemPaletteMono[] = {1, 0, 1, 1};
+color_t defaultTitlePaletteMono[] = {0, 1, 0, 0};
+
+#define TITLE_PADDING 2
+#define TITLE_SPACING 2
+
+/**
+ * This is one of the stock themes, you can modify it to meet your requirements, and it will not be updated by tcMenu
+ * Designer unless you delete it. This sets up the fonts, spacing and padding for all items.
+ * @param gr the graphical renderer
+ * @param itemFont the font for items
+ * @param titleFont the font for titles
+ * @param needEditingIcons if editing icons are needed
+ */
+void installMonoInverseTitleTheme(GraphicsDeviceRenderer& gr, const MenuFontDef& itemFont, const MenuFontDef& titleFont,
+                                  bool needEditingIcons, BaseGraphicalRenderer::TitleMode titleMode, bool useUnicode) {
+    TcThemeBuilder themeBuilder(gr);
+    themeBuilder.withSelectedColors(0, 2)
+            .dimensionsFromRenderer()
+            .withItemPadding(MenuPadding(1))
+            .withRenderingSettings(titleMode, false)
+            .withPalette(defaultItemPaletteMono)
+            .withNativeFont(itemFont.fontData, itemFont.fontMag)
+            .withSpacing(1);
+
+    if(needEditingIcons) {
+        themeBuilder.withStandardLowResCursorIcons();
+    }
+
+    if(useUnicode) {
+        themeBuilder.enablingTcUnicode();
```

Comment:
Just a side question: Why do we have api like `enabling...Unicode`? I think that `enableTccUnicode` makes more sense.